### PR TITLE
PropTypes to flow annotations

### PR DESCRIPTION
**Pull request by amensouissi**

### CODE REVIEW

File: assembl/static2/js/app/components/debate/survey/post.jsx [hunk 1]
```diff
@@ -233,6 +233,7 @@ class Post extends React.Component<Props> {
     return (
       <div className="shown box" id={post.id}>
         <div className="content">
+          {/* $FlowFixMe */}
```
Comment (MilanBarande) [inline assembl/static2/js/app/components/debate/survey/post.jsx]: What's the issue here ?

File: assembl/static2/js/app/pages/debate.jsx [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 import Themes from '../components/debate/common/themes';
 import DebateThematicsQuery from '../graphql/DebateThematicsQuery.graphql';
 
-const Debate = ({ identifier, data, params, children }) => {
+type Props = {
+  identifier: string,
+  data: {
+    loading: boolean,
+    error: ?Error,
+    thematics: DebateThematicsQueryQuery
```
Comment (MilanBarande) [inline assembl/static2/js/app/pages/debate.jsx]: QueryQuery ?

Comment (amensouissi) [reply assembl/static2/js/app/pages/debate.jsx]: it's a generated name: see the flow/graphql_types.flow.js

File: assembl/static2/js/app/components/debate/survey/post.jsx (hunk 1 above)
Comment (amensouissi) [reply assembl/static2/js/app/components/debate/survey/post.jsx]: the creatorName cannot be null

**Pull request merged**